src/main/java/net/snowflake/client/jdbc/SnowflakeColumnMetadata.java

```diff
@@ -194,6 +197,11 @@ public void setAutoIncrement(boolean autoIncrement) {
     isAutoIncrement = autoIncrement;
   }
 
+  @SnowflakeJdbcInternalApi
```

Comment:
Why is internal? I believe that this is something that we want to expose?

Reply:
The whole class should be internal, we also have the internal constructor now